src/telemetry_poller_vm.erl

```diff
@@ -0,0 +1,35 @@
+-module(telemetry_poller_vm).
+
+-export([
+  memory/0,
+  total_run_queue_lengths/0
+]).
+
+-spec memory() -> ok.
+memory() ->
+  Measurements = erlang:memory(),
+  telemetry:execute([vm, memory], maps:from_list(Measurements), #{}).
+
+-spec total_run_queue_lengths() -> ok.
+total_run_queue_lengths() ->
+  Total = cpu_stats(total),
+  CPU = cpu_stats(cpu),
+  telemetry:execute([vm, total_run_queue_lengths], #{
+    total => Total,
+    cpu => CPU,
+    io => Total - CPU},
+    #{}).
+
+-ifdef(OTP_RELEASE).
```

Comment:
So according to the docs, OTP_RELEASE was introduced on Erlang 21. So here is my suggestion:

```erlang
-ifdef(OTP_RELEASE).
  -spec cpu_stats(total | cpu) -> non_neg_integer().
    cpu_stats(total) ->
      erlang:statistics(total_run_queue_lengths_all);
    cpu_stats(cpu) ->
      erlang:statistics(total_run_queue_lengths).
-else.
  -spec cpu_stats(total | cpu) -> non_neg_integer().
  cpu_stats(_) ->
    lists:sum(erlang:statistics(run_queue_lengths)).
-endif.

Data will be a bit skewed on 20 but i think it is fine.